### DISCUSSION TITLE
Template matcher testing beginning of the document

### DISCUSSION THREAD

**Discussion by Michal-Mikolas** [projectdiscovery/nuclei-templates#9375] [Q&A] [answered]
How can I create matcher that tells Nuclei to check `Returned content must START with the text "SQLite"` ? 

I mean not only contain "SQLite" somewhere, it needs to be the first word of the document.  
And not `some line needs to start with "SQLite"`, but the FIRST line needs to start with it. 

There is no mention about this in the [documentation](https://github.com/projectdiscovery/nuclei-docs/blob/main/docs/templating-guide/operators/matchers.md).

**Comment by ehsandeep** [comment, marked as answer]
You can use `startswith` [dsl function](https://github.com/projectdiscovery/dsl), for example - https://github.com/projectdiscovery/nuclei-templates/pull/9370/files#diff-95db2908ad8b334ba6d338376097135a09c51b12b7329f3b08b44ed23783318eR21

**Reply by Michal-Mikolas** [reply]
Wow this is very powerfull :-O Thanks.
